internal/commands/user/delete/inputs.go

```diff
@@ -0,0 +1,79 @@
+package delete
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/commands/user/shared"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type inputs struct {
+	cli.ProjectAppInputs
+	shared.UsersInputs
+	Users []string
+}
+
+func (i *inputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	if err := i.ProjectAppInputs.Resolve(ui, profile.WorkingDirectory); err != nil {
+		return err
+	}
+	// Interactive set Status
+	if i.Status == shared.StatusTypeInteractive {
```

Comment:
do you think its possible for us to ignore any "interactive mode" for now, and implement that in a separate ticket?  im ok with this command being temporarily verbose while we figure out how we want to go about allowing these interactive-by-default flags to work

Reply:
Removed interactive flag modes in next commit